# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1420437169 -3600
#      Mon Jan 05 06:52:49 2015 +0100
# Branch gui-release
# Node ID 9da61e3ecbe7aa72f5972b83d38a7e89906d749c
# Parent  52c51472b6b9d622ace8272886c795b4a8046433
backout cset c2d01ed114ba

diff --git a/libgui/src/m-editor/find-dialog.h b/libgui/src/m-editor/find-dialog.h
--- a/libgui/src/m-editor/find-dialog.h
+++ b/libgui/src/m-editor/find-dialog.h
@@ -74,19 +74,20 @@ class QPushButton;
 class find_dialog : public QDialog
 {
   Q_OBJECT
 public:
   find_dialog (QsciScintilla* edit_area, QWidget *parent = 0);
   void init_search_text ();
 
 private slots:
-
+#ifdef HAVE_QSCI_FINDSELECTION
   void handle_sel_search_changed (int);
-  void handle_selection_changed (bool);
+  void handle_selection_changed (bool has_selected);
+#endif
   void handle_backward_search_changed (int);
   void handle_search_text_changed (QString new_search_text);
 
   void find (bool forward = true);
   void find_next ();
   void find_prev ();
   void replace ();
   void replace_all ();
